test/test_integration.py

```diff
@@ -980,32 +980,49 @@ def test_gallery(self):
         dt_input_1.click()
         self.snapshot('gallery - DatePickerSingle\'s datepicker '
                       'when no date value and no initial month specified')
+        dt_length = len(dt_input_1.get_attribute('value'))
+        dt_input_1.send_keys(dt_length * Keys.BACKSPACE)
         dt_input_1.send_keys("1997-05-03")
 
         dt_input_2 = self.driver.find_element_by_css_selector(
             '#dt-single-no-date-value-init-month #date'
         )
+        self.driver.find_element_by_css_selector(
+            'label'
+        ).click()
         dt_input_2.click()
         self.snapshot('gallery - DatePickerSingle\'s datepicker '
                       'when no date value, but initial month is specified')
+        dt_length = len(dt_input_2.get_attribute('value'))
+        dt_input_2.send_keys(dt_length * Keys.BACKSPACE)
         dt_input_2.send_keys("1997-05-03")
 
         dt_input_3 = self.driver.find_element_by_css_selector(
             '#dt-range-no-date-values #endDate'
         )
+        self.driver.find_element_by_css_selector(
+            'label'
+        ).click()
         dt_input_3.click()
         self.snapshot('gallery - DatePickerRange\'s datepicker '
                       'when neither start date nor end date '
                       'nor initial month is specified')
+        dt_length = len(dt_input_3.get_attribute('value'))
+        dt_input_3.send_keys(dt_length * Keys.BACKSPACE)
         dt_input_3.send_keys("1997-05-03")
 
         dt_input_4 = self.driver.find_element_by_css_selector(
             '#dt-range-no-date-values-init-month #endDate'
         )
+        self.driver.find_element_by_css_selector(
+            'label'
+        ).click()
```

Comment:
Now that we are passing a valid value to react-dates, the calendar opens -- the alternative is throwing an error and crashing the render, not great.. all this means it's in the way when trying to select another component. Clicking something safe to close it.